src/shared/templates/shared/partial/popover_tour.html

```diff
@@ -0,0 +1,384 @@
+            <li><span class="feature-icon icon-switch" aria-hidden="true"></span> Translate or simplify</li>
+            <li><span class="feature-icon icon-play-circled2" aria-hidden="true"></span> Hear the words read aloud</li>
+        </ul>
+        <div class="row gx-0_5 flex-items-center">
+            <div class="col">
+                {% if has_teacher_resource %}
+                    {% if clusive_user.role == 'TE' or clusive_user.role == 'PA' %}
+                        <!-- 'context' type popovers do not have a "Learn more" resource link yet, so this is blank -->
+                        <a href="#" tabindex="-1"></a>
+                    {% endif %}
+                {% endif %}
+            </div>
+            <div class="col-auto tooltip-action">
+                <button type="button" class="btn btn-primary" data-cfw-dismiss="popover">Got it!</button>
+            </div>
+        </div>
+        {% elif tour_name == "view" %}
+<!-- view -->
+        <div class="embed-fluid embed-fluid-16x9 mb-0_5">
+            <iframe id="viewIframe" src="https://videos.sproutvideo.com/embed/ac9edbbb141fe6c525/86b0cbe36e2ab800?type=hd" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen aria-label="Library Views video"></iframe>
+        </div>
+        <p>View the Clusive library by all readings, or in smaller groupings like class readings, readings you’ve starred, public readings, or readings you’ve uploaded.</p>
+        <div class="row gx-0_5 flex-items-center">
+            <div class="col">
+                {% if has_teacher_resource %}
+                    {% if clusive_user.role == 'TE' or clusive_user.role == 'PA' %}
+                        <!-- 'view' type popovers currently do not have a "Learn more" resource link yet, so this is blank -->
+                        <a href="#" tabindex="-1"></a>
+                    {% endif %}
+                {% endif %}
+            </div>
+            <div class="col-auto">
+                <button type="button" class="btn btn-primary" data-cfw-dismiss="popover">Got it!</button>
+            </div>
+        </div>
+        {% elif tour_name == "book_actions" %}
+<!-- book_actions -->
+        {% if clusive_user.role == 'TE' %}
+        <div class="embed-fluid embed-fluid-16x9 mb-0_5">
+            <iframe id="book_actionsIframe" src="https://videos.sproutvideo.com/embed/799edbb41512e5c4f0/ba8fab36c9170dc4?type=hd" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen aria-label="Book actions video"></iframe>
+        </div>
+        {% endif %}
+        <p>Click “More actions” (three dots) to assign or customize any book, and for books you uploaded to Clusive, to edit information or delete a book.</p>
+        <div class="row gx-0_5 flex-items-center">
+            <div class="col">
+                {% if has_teacher_resource %}
```

Comment:
The general rationale of `has_teacher_resource` was flexibility in showing the link.  It should have been named `has_teacher_resource_link` or, even better, `show_teacher_resource_link`.

Admittedly it's somewhat muddled in that the link is not shown for student and guest users and that could be handled by `show_teacher_resource_link`.  But the template covers that situation by the check for a TE or PA role*.

Still, there are cases where the user is a teacher or parent and the link should not be shown.  For example, if the user has navigated to the Manage page and the Manage popover shows, the link in the popover is odd since it just navigates to the page the user is viewing.  It should not be shown in this case.

As for the Library case, that's a bug; it should not be false.

* - Perhaps this is part of the refactoring:  Move the role check logic to `views.py` to set  `show_teacher_resource_link` and then use only that boolean in the template.  It would remove some clutter.